src/__tests__/linter.test.ts

```diff
@@ -642,7 +642,7 @@ responses:: !!foo
       }),
       expect.objectContaining({
         code: 'oas3-valid-schema-example',
-        message: '"foo.example" property type should be number',
+        message: 'property type should be number',
```

Comment:
@philsturgeon is this something you expected to see?
I believe there is no point in printing paths when it exists in the document.
The only reasonable case is to print them when the property **does not** exist in the document and you want to indicate that, i.e. "XYZ property is not truthy".

We could keep it configurable though and let users decide, but leaving the call up to you all.

cc @marbemac

Reply:
I would expect to see "example property type should be number" or `"The property type of \"example\" should be number."`

Reply:
I agree that having just that little bit more context in the error message is helpful. Suggest we keep it in the pretty message.